interfaces/builtin/serial_port_test.go

```diff
@@ -261,6 +288,15 @@ SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{idVendor}=="ffff", ATTRS{idProduct}==
 	c.Assert(spec.Snippets(), HasLen, 1)
 	snippet = spec.Snippets()[0]
 	c.Assert(snippet, Equals, expectedSnippet2)
+
+	spec = &udev.Specification{}
+	expectedSnippet3 := `IMPORT{builtin}="usb_id"
+SUBSYSTEM=="tty", SUBSYSTEMS=="usb", ATTRS{idVendor}=="abcd", ATTRS{idProduct}=="1234", ENV{ID_USB_INTERFACE_NUM}=="00", SYMLINK+="serial-port-myserial"`
```

Comment:
Same question, why `00` and not just `0`?

Reply:
I just checked the ID_USB_INTERFACE_NUM is set to`00`, it must be '00' or it cannot match this attribute.(I have manually tried).

Reply:
That's all right, can you please add a comment stating as such next to the code that generates this snippet please?

Reply:
Added!